courses/free-courses-es.md

```diff
@@ -146,6 +148,11 @@
 * [Word Avanzado (2010)](https://www.pildorasinformaticas.es/course/word-avanzado-2010) - Juan Díaz (Píldoras Informáticas)
 
 
+### Patrones de diseño
+
+* [Patrones de diseño](https://refactoring.guru/es/design-patterns) - Alexander Shvets (Refactoring.Guru)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
belongs in Programación I think

Reply:
You are right. I have already placed this resource in the programming category